tests/contrib/funsor/test_enum_funsor.py

```diff
@@ -1390,3 +1390,73 @@ def guide_iplate():
             elbo = infer.TraceEnum_ELBO(max_plate_nesting=1)
             actual_loss = elbo.differentiable_loss(model_iplate_iplate, guide_iplate)
     _check_loss_and_grads(expected_loss, actual_loss)
+
+
+@pyroapi.pyro_backend(_PYRO_BACKEND)
+def test_elbo_enumerate_plate_9():
```

Comment:
Can you add a couple more tests that work along similar lines (comparing ELBO values in models with parallel and sequential plates)? e.g. some of the following:
```
a -> ( (cij) )

( ai -> ( bij ) )

a -> ( bi -> ( cij ) )

a -> ( bi )
|
V
( cj )
```
You might be able to copy some of these from existing tests.